packages/codemods/rollup.config.mjs

```diff
@@ -0,0 +1,28 @@
+import babel from '@rollup/plugin-babel';
```

Comment:
FYI: Assuming we run codemods through the `jscodeshift` CLI, we might only need rollup for testing purposes. The `jscodeshift` CLI will handle TS files JIT.

Reply:
Indeed, was able to remove this.